packages/fxa-event-broker/lib/notificationProcessor.ts

```diff
@@ -118,17 +126,19 @@ class ServiceNotificationProcessor {
 
   private async handleMessage(sqsMessage: SQS.Message) {
     const body = JSON.parse(sqsMessage.Body || '{}');
-    const message = joi.attempt(JSON.parse(body.Message), BASE_MESSAGE_SCHEMA);
+    const message = joi.attempt(body, BASE_MESSAGE_SCHEMA);
     switch (message.event) {
       case LOGIN_EVENT: {
         const loginMessage = joi.attempt(message, LOGIN_SCHEMA);
         await this.db.storeLogin(loginMessage.uid, loginMessage.clientId);
+        this.logger.debug('sqs.loginEvent', loginMessage);
         return;
       }
       case SUBSCRIPTION_UPDATE_EVENT: {
```

Comment:
I'm super fuzzy on SQS goings-on in FxA right now - is this `SUBSCRIPTION_UPDATE_EVENT` something yet to be implemented on the `fxa-auth-server` side? I seem to remember we do a [`push.notifyProfileUpdated(uid, devices);` and `log.notifyAttachedServices('profileDataChanged', request, { uid, email });`](https://github.com/mozilla/fxa/blob/master/packages/fxa-auth-server/lib/routes/subscriptions.js#L160) when subscriptions are created and deleted. But, I'm not sure if that does the trick here.

Reply:
There's a separate event emitted for subscription state changes that @philbooth landed awhile back.